tgui/packages/tgui/interfaces/TeleConsole/index.tsx

```diff
@@ -6,63 +6,131 @@
+                onClick={() => setTab(Tab.Local)}
+              >
+                Local
+              </Tabs.Tab>
+              <Tabs.Tab
+                icon="globe"
+                selected={tab === Tab.LongRange}
+                onClick={() => setTab(Tab.LongRange)}
+              >
+                Long Range
+              </Tabs.Tab>
+            </Tabs>
+          </Stack.Item>
+
+          {(tab===Tab.Local) && (
+            <Stack.Item>
+              <Section>
+                <CoordinatesSection />
+                <Section>
+                  <Button icon="sign-out-alt" onClick={() => act('send')} disabled={!isConnectedToHost}>
+                    Send
+                  </Button>
+                  <Button icon="sign-in-alt" onClick={() => act('receive')} disabled={!isConnectedToHost}>
+                    Receive
+                  </Button>
+                  <Button onClick={() => act('portal')} disabled={!isConnectedToHost}>
+                    <Icon name="ring" rotation={90} />
+                    Toggle Portal
+                  </Button>
+                  <Button icon="magnifying-glass" onClick={() => act('scan')} disabled={!isConnectedToHost}>
+                    Scan
+                  </Button>
+                </Section>
+                {readout && <Section>{formatReadout(readout)}</Section>}
+                <BookmarksSection
+                  bookmarks={bookmarks}
+                  maxBookmarks={maxBookmarks}
+                  onAddBookmark={handleAddBookmark}
+                  onDeleteBookmark={handleDeleteBookmark}
+                  onRestoreBookmark={handleRestoreBookmark}
+                  targetCoords={[xTarget, yTarget, zTarget]}
+                />
+                <ConnectionSection
+                  isConnected={isConnectedToHost}
+                  isPanelOpen={!!isPanelOpen}
+                  onCyclePad={handleCyclePad}
+                  onReset={handleResetConnect}
+                  onRetry={handleRetryConnect}
+                  padNum={padNum}
+                />
+              </Section>
+            </Stack.Item>
+          )}
+          {(tab===Tab.LongRange) && (
+            <Stack.Item>
+              <Section>
+                <LongRangeSection
+                  isConnected={isConnectedToHost}
+                  destinations={destinations}
+                  onSend={handleLongRangeSend}
+                  onReceive={handleLongRangeReceive}
+                  onToggle={handleLongRangePortal}
+                />
+                {readout && <Section>{formatReadout(readout)}</Section>}
+                <ConnectionSection
+                  isConnected={isConnectedToHost}
+                  isPanelOpen={!!isPanelOpen}
+                  onCyclePad={handleCyclePad}
+                  onReset={handleResetConnect}
+                  onRetry={handleRetryConnect}
+                  padNum={padNum}
+                />
+              </Section>
```

Comment:
Wrapping all of these in a `Section` is (probably) not needed. `Section`s one after the other *should* look fine without a wrapping `Section`.

Reply:
It is needed, or at least a means, to maintain a consistent with the existing look and feel where there is no visual break between sections.
Implemented -> Suggested  and the Local page.
![image](https://github.com/goonstation/goonstation/assets/1017374/2a45bdee-7f85-4cf9-b617-0f68349493bb)

Reply:
Gotcha. I don't actually have issue with the second one, as the connection status isn't really part of the Destinations section so it being its own visual thing seems "fine" to me, but I'm not going to nitpick it further.